gems/pending/spec/appliance_console/logfile_configuration_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "appliance_console/logfile_configuration"
+
+describe ApplianceConsole::LogfileConfiguration do
+  before do
+    @spec_name = File.basename(__FILE__).split(".rb").first.freeze
+    allow(LinuxAdmin::Service).to receive(:new).and_return(double(@spec_name, :running? => false))
+    allow(subject).to receive(:clear_screen)
+    allow(subject).to receive(:say)
+  end
+
+  describe "#ask_questions" do
+    context "when user confirms a new disk" do
```

Comment:
You can collapse this whole context into one `it "returns true when user confirms a new disk" do` block.

I don't think the extra lines and separating out the setup really help if there is only one spec.